configs/styleganv2/stylegan2_c2_8xb4_ffhq-1024x1024.py

```diff
@@ -42,7 +42,7 @@
                                                         0.99**d_reg_ratio))))
 
 batch_size = 4
-data_root = './data/ffhq/images'
+data_root = 's3://openmmlab/datasets/editing/ffhq/images'
```

Comment:
Please keep this config unchanged since this path only works in the internal network. 
Noted that, you can generate configs with *ceph path* by `.dev_scripts/create_ceph_configs.py`